CryptoAnalysis/src/main/java/crypto/reporting/CSVReporter.java

```diff
@@ -190,6 +190,8 @@ private void writeToFile() {
 				line.add(string);
 			}
 			writer.write(Joiner.on(CSV_SEPARATOR).join(line) + "\n");
+			writer.write("\n"+REPORT_NAME.substring(0,14)+"\n");
```

Comment:
14 looks very concrete to me. Is is save to use?

Reply:
I changed this and also how the version is loaded.